conduce.py

```diff
@@ -338,10 +379,17 @@ def main():
     parser_config_find.add_argument('--name', help='The name of the dataset to query')
     parser_config_find.add_argument('--id', help='The ID of the dataset to query')
     parser_config_find.add_argument('--regex', help='An expression to match datasets and query')
-    parser_config_find.add_argument('--content', help='Content to retreive: id,full,meta')
+    parser_config_find.add_argument('--content', help='Content to retrieve: id,full,meta')
     parser_config_find.add_argument('--decode', action='store_true', help='Decode base64 and JSON for full content requests')
     parser_config_find.set_defaults(func=find_resource)
 
+    parser_config_edit = subparsers.add_parser('edit',  parents=[api_cmd_parser], help='edit resources that match the given parameters')
+    parser_config_edit.add_argument('--type', help='Conduce resource type to edit')
+    parser_config_edit.add_argument('--name', help='The name of the dataset to query')
+    parser_config_edit.add_argument('--id', help='The ID of the dataset to query')
```

Comment:
The help messages should refer to `resource` instead of `dataset`.

Reply:
Good catch.  Thanks.